src/main/webapp/app/shared/monaco-editor/monaco-diff-editor.component.ts

```diff
@@ -143,7 +153,7 @@ export class MonacoDiffEditorComponent implements OnInit, OnDestroy {
      * @param artemisTheme The active Artemis theme.
      */
     changeTheme(artemisTheme: Theme): void {
-        monaco.editor.setTheme(artemisTheme === Theme.DARK ? 'vs-dark' : 'vs-light');
+        this.monacoEditorService.applyTheme(artemisTheme);
```

Comment:
you could also replace the theme subscription with a signal (which doesn't have to be unsubscribed)


```suggestion
        private theme = toSignal(this.themeService.getCurrentThemeObservable(), { requireSync: true });

        constructor() {
            effect(() => {
                this.changeTheme(this.theme());
            });
        }
```

Reply:
Done